### ISSUE TITLE
Guild charter not working!

### ISSUE THREAD

**Issue by jasond12** [LightsHope/server#1054]
I have been trying to create a guild - quite simply for fun and PvP, although I'm having issues creating one, I can't even obtain one signature (using my own alts on another account for testing purposes)

I purchase a guild charter and give it a name, it then appears in my bag, I place the charter in my original first bag as this has solved a lot of issues, but sadly without success this time. I then open it and request some signatures from people, some players kindly decline and some try to sign the charter. The ones who try to sign it - get the message 'You have already signed that charter'. It's weird because after creating a level 1 alt and trying to sign my charter on my main, it gives me that message, when the char hasn't signed any charters given the toon was created barely a minute ago. I have deleted the charter and purchased it again with the same results, some people are having this issue also

**Comment by deftproxy** [comment]
Same problem here

**Comment by kevinmd88** [comment]
Also having this issue. Not much I can offer though. I created a guild charter through the guild vendor and went to ask a few of my friends to sign it. I can tell you that I tried doing so with a friend who was on my own LAN with me (same WAN IP address), and with a few others who weren't. I even went so far as to have them all create new accounts to see if anything was blocking it, but even with brand-new accounts and freshly-made characters it was presenting this message.

My charter as it stands now has no signatures because I have not encountered an instance where it worked. I have not tried to create a new guild charter yet though, but OP's comments suggest the issue would persist. I will still attempt to try it.
